pkg/policy/cidr.go

```diff
@@ -68,3 +68,26 @@ func GetCIDRPrefixes(rules api.Rules) []*net.IPNet {
 	}
 	return res
 }
+
+// GetCIDRPrefixesFromIPs returns all of the ips as a slice of *net.IPNet.
+func GetCIDRPrefixesFromIPs(ips []net.IP) []*net.IPNet {
```

Comment:
This could probably be put into `pkg/ip`.

Reply:
Yes please, this looks like exactly the kind of function that can end up existing in several places in the codebase if we don't put it somewhere generic or don't go looking for it :-)